core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Subproject.java

```diff
@@ -42,4 +42,8 @@
 
   void setPriority(Boolean priority);
 
+  void setReferenceGenomeId(Long referenceGenomeId);
+
+  Long getReferenceGenomeId();
```

Comment:
Here in Subproject the Long object is used but in Project the primitive long is used. Should be consistent unless there's a specific reason.